partials/tabs/projects.html

```diff
@@ -6,10 +6,7 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="text" class="validate">
       </div>
       <div class="input-field col-md-2">
```

Comment:
@hemangsk here u go!! It remains the same though.
should i commit these changes as well ?
![screenshot from 2017-01-31 00-31-34](https://cloud.githubusercontent.com/assets/13048113/22437103/b39fe28a-e74c-11e6-97e7-b182104c639f.png)